docs/internal/contract-inbound-webhook.md

```
@@ -0,0 +1,147 @@
+|---|---|
+| HTTP method | `POST` |
+| Path | `/wp-json/swh/v1/inbound-email` |
+| Registration | `register_rest_route( 'swh/v1', '/inbound-email', … )` |
+| Handler | `swh_handle_inbound_email( WP_REST_Request $request )` in `simple-wp-helpdesk/includes/class-email.php` |
+| Permission callback | Returns `true` — **auth is enforced inside the handler** via `hash_equals()` against the bearer header |
+
+---
+
+## Authentication
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)